Makefile

```diff
@@ -538,7 +538,7 @@ log-update-translations:
 update-translations: | log-update-translations
 	cmake -S $(TS_SOURCE_DIR) -B $(TS_BUILD_DIR) -Wno-dev $(HANDLE_OUTPUT)
 	cmake --build $(TS_BUILD_DIR) --target update_application_translations $(HANDLE_OUTPUT)
-#	+ cd scripts/translationScripts && ./fixup-base-ts-for-lokalise.py $(HANDLE_OUTPUT)
+	+ cd scripts/translationScripts && go run fixup-base-ts-for-lokalise.go $(HANDLE_OUTPUT)
```

Comment:
The fist of this change; we now actually run the "fixup" script as part of the build, not just manually